### PR TITLE
Awesome Book with ES6

### PR DESCRIPTION
**Pull request by robinbenitezmora**
In order to understand how ES6 works, was designed this project follows the new rules by coding any future project in Javascript.

### CODE REVIEW

**Review by Bondok6 — COMMENTED** [review]
## Hi @robinbenitezmora 🙋‍♂️, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Linter errors

You have submitted a project with the wrong Linter setup, kindly add `.github/workflows` folders and `linters.yml` file to setup Linters correctly, I think you forgot to add `.github` folder
1. Linter should be set up correctly for this repository.
1. Linter result should be green, this means that the last commit have a green check mark (✅)
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        <li><a href="#" id="list">List</a></li>
+        <li class="separate"></li>
+        <li><a href="#" id="add-book">Add Book</a></li>
+        <li class="separate"></li>
+        <li><a href="#" id="contact">Contact</a></li>
+      </ul>
+    </nav>
+  </header>
+  <p id="date" class="time"></p>
+  <h2 class="title_list">All awesome books</h2>
```
Comment (Nemwel-Boniface) [inline index.html:24]: - [ ] [OPTIONAL] Kindly note that this text should not be visible when clicking on all pages. Kindly consider hiding it when the user clicks on another page.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  const minute = date.getMinutes();
+  const second = date.getSeconds();
+  const time = `${month} ${day} ${year}, ${hour}:${minute}:${second}`;
+  showTime.innerText = time;
+};
+
+const startTime = () => {
+  updateTime();
+  setInterval(updateTime, 1000);
+};
```
Comment (Nemwel-Boniface) [inline index.js]: - [ ] According to the project requirements, you are required to use the [Luxon](https://moment.github.io/luxon/#/install?id=es6) library for your time and data and not a custom function for this section. Kindly include the Luxon library and use it to generate your date and time details.

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @robinbenitezmora            :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

You have done an amazing job with your Awesome books project :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 2 above)
Comment (FrederickMih) [inline index.js]: - [ ] As requested by the previous reviewer, you are supposed to use the [Luxon](https://github.com/moment/luxon/blob/master/docs/install.md) library. The script for this library should be downloaded and saved it as a file in your project directory(full or minified version) :+1: . Next, import it as a module in your index.js file. [This article](https://github.com/moment/luxon/blob/master/docs/tour.md#get-the-current-date-and-time) would assist you to set it up.
- Note that you have added the library :+1: but you didn't import it. Let's kindly use the library.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @robinbenitezmora , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

# To Highlights

- Correct use of ES6 modules :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @robinbenitezmora 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**